.devcontainer/Dockerfile

```diff
@@ -2,5 +2,13 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+RUN apt update; \
+    apt install -y software-properties-common make g++; \
+    add-apt-repository ppa:deadsnakes/ppa; \
+    apt update; \
+    apt install python3.12; \
+    ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
- use `install-tool python 3.12.x` like yarn below 
- why make and g++?

Reply:
make and g++ are needed because re2 package is probably not available on ARM.  Those tools are used for building the package while running `yarn install`.